src/Viewer.js

```diff
@@ -76,6 +77,14 @@ export class Viewer {
         this.mouseDownPosition = new THREE.Vector2();
         this.mouseDownTime = null;
 
+        this.pointerUpHandler = this.onMouseUp.bind(this);
+
+        this.loadingSpinner = new LoadingSpinner();
+        this.loadingSpinner.hide();
+
+        this.viewerNeedsUpdate = true;
```

Comment:
Could we call this flag `viewNeedsRefresh` ? Just so it's clear that just the view/screen needs to be updated.

Reply:
Sure, I was just kind of going for the "viewMatrixNeedsUpdate"-feel, but I don't have any hard stance on either